code/game/area/areas.dm

```diff
@@ -88,7 +88,7 @@
 	///Used to decide what the minimum time between ambience is
 	var/min_ambience_cooldown = 30 SECONDS
 	///Used to decide what the maximum time between ambience is
-	var/max_ambience_cooldown = 90 SECONDS
+	var/max_ambience_cooldown = 60 SECONDS //SKYRAT EDIT CHANGE - ORIGINAL: 90
```

Comment:
So, after spending quite a lot of time in-game since that was test-merged, I can safely say that 60 seconds is not a long-enough cooldown. Ambience is overlapping waaaaay too often. I'm not sure if you want me to make a quick PR to revert this or if you'd rather do it yourself.